numpy/random/mtrand/mtrand.pyx

```diff
@@ -2524,11 +2524,11 @@ cdef class RandomState:
         the probability density function:
 
         >>> import matplotlib.pyplot as plt
-        >>> import scipy.special as sps
-        >>> count, bins, ignored = plt.hist(s, 50, normed=True)
-        >>> x = np.arange(-np.pi, np.pi, 2*np.pi/50.)
-        >>> y = -np.exp(kappa*np.cos(x-mu))/(2*np.pi*sps.jn(0,kappa))
-        >>> plt.plot(x, y/max(y), linewidth=2, color='r')
+        >>> from scipy.special import i0
+        >>> plt.hist(s, 50, normed=True)
+        >>> x = np.linspace(-np.pi, np.pi, num=51)
+        >>> y = np.exp(kappa*np.cos(x-mu))/(2*np.pi*i0(kappa))
+        >>> plt.plot(x, y, linewidth=2, color='r')
```

Comment:
I'm guessing that the curve was supposed to pass through the tops of the histogram bars. Might be worth running a check...